src/AlertDestination/Mail.php

```diff
@@ -51,7 +51,7 @@ public function setParameters(array $parameters)
      */
     public function trigger(Alert $alert)
     {
-        if (!getenv('MAILER_FROM')) {
+        if (!$_ENV['MAILER_FROM']) {
```

Comment:
The [getenv](https://www.php.net/manual/en/function.getenv.php) function returns false if the environment variable doesn't exist, accessing the `$_ENV` superglobal directly doesn't have this feature. So, this change can cause undefined index errors.

```suggestion
        $mailerFrom = $_ENV['MAILER_FROM'] ?? null;
        if (!$mailerFrom) {
```